### DISCUSSION TITLE
How to change the Enterprise overview site via API?

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#117225] [Enterprise]
A hopefully simple question... ;)

**Comment by queenofcorgis** [comment]
Hi there @Timmmy-nlb, 

I hope it is simple too 😃 - do you mind elaborating on what you mean by overview site so the community can better assist?

**Reply by Timmmy-nlb** [reply]
oh... I'm sorry if I was too vague.

I would like to be able to update this overview page via API:
![grafik](https://github.com/community/community/assets/52195132/7bd6dba7-ebde-4950-b7f7-09de338601af)

**Comment by queenofcorgis** [comment]
@Timmmy-nlb at this this time I'm not seeing an API for changing this. If you'd like to update your post to give product feedback (feel free to include desired state, use cases, etc. I'll change the label to `Product Feedback` so it can be queued for appropriate team.

**Comment by Timmmy-nlb** [comment]
Product feedback
Funktion: Enterprise Overview - readme - Site

Hello, the new function of the overview page for GitHub Enterprise is a good thing, also good that it can be read by people who have not yet been assigned a license (EMU).

What I'm still missing is a way to change or update this page via API.
That would help me a lot, because the manual customization could be quite frequent and therefore perfect for automation.

Thank you

**Reply by queenofcorgis** [reply]
Thanks for sharing - I added the label 🏷️
